spec/dishes_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'dishes'
+
+describe Dishes do
+
+    let(:my_dishes) { Dishes.new }
+
+    describe "#create_menu" do
+        it "Should add a new dish to the menu with a name and price" do
+            price = 8.95
```

Comment:
Would it be clearer to write name = "Chicken Tikka" here and pass both variables in below?